### DISCUSSION TITLE
Parallel testing

### DISCUSSION THREAD

**Discussion by d3vl1m3** [oblador/loki#383] [Q&A] [answered]
Heya - I was wondering what the speed of Loki + Storybook was like compared to Backstop JS. 

We are using BSJS at work but want to consolidate our code to use SB and a VRT framework such as Loki.

I'm aware that BSJS can take and compare multiple images at once with its `asyncCaptureLimit` and `asyncComapreLimit` flags. Does Loki do something similar? I can't see anything about this in the docs.

Any insight is appreciated :)

**Comment by oblador** [comment, marked as answer]
Yep, you can use `--chromeConcurrency` https://loki.js.org/command-line-arguments.html

**Reply by d3vl1m3** [reply]
Amazing. I was looking at the configuration and didn't even think to look at the CLI section 🤦 

Is there a way that some of these config options can be ported in to a JSON format that can be set up in its own file or in package.json?

**Reply by oblador** [reply]
Yep, either package.json or loki.config.js should work!

**Reply by d3vl1m3** [reply]
Ah! I see it now. Didn't realise you could just add the to the top level of the config 🥳
